features/release.feature

```diff
@@ -367,6 +367,36 @@ MARKDOWN
           ASSET_TARBALL
           """
 
+  Scenario: Delete a release
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/will_paginate/releases') {
+          json [
+            { url: 'https://api.github.com/repos/mislav/will_paginate/releases/123',
+              upload_url: 'https://api.github.com/uploads/assets{?name,label}',
+              tag_name: 'v1.2.0',
+              name: 'will_paginate 1.2.0',
+              draft: true,
+              prerelease: false,
+              assets: [
+                { url: 'https://api.github.com/repos/mislav/will_paginate/assets/9876',
```

Comment:
None of it was that hard. I just wasn't familiar with how things worked. Specifically that I was setting up the JSON object to be returned and the acting on it with the second command. As with all things, once I understood what it was looking for it made more sense.